test/instrumentation/modules/http2.js

```diff
@@ -27,7 +27,7 @@ isSecure.forEach(secure => {
 
     // Note NODE_OPTIONS env because it sometimes has a setting relevant
     // for this test.
-    t.comment(`NODE_OPTIONS=${process.env.NODE_OPTIONS}`)
+    t.comment(`NODE_OPTIONS=${process.env.NODE_OPTIONS || ''}`)
```

Comment:
REVIEW NOTE: Otherwise this emits a test comment that `NODE_OPTIONS=undefined`, which is slightly misleading.